microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md

```diff
@@ -55,11 +55,13 @@ To edit (or define) ATP policies, you must be assigned an appropriate role. The
 
 4. Specify a name and description for your policy.
 
-5. In the **Do not rewrite the following URLs** section, select the **Enter a valid URL** box, and then type a URL, and then choose the plus sign (+).
+5. Turn **ON** URLs will be rewritten and checked against a list of known malicious links when user clicks on the link.
```

Comment:
```suggestion
5. Turn **ON** URLs will be rewritten and checked against a list of known malicious links when the user clicks on the link.
```

Reply:
I am really not sure about this. Please confirm.

Reply:
This is a setting in o365 and it states "URLs will be rewritten and checked against a list of known malicious links when user clicks on the link." We are turning it ON. So please confirm if this change is required.

Reply:
Thank you for the explanation. Now it makes sense. Please ignore my suggestion.